### PR TITLE
Sticky posts

### LINKED ISSUE #13: Add ability to make posts sticky

### PR DESCRIPTION
**Pull request by takeontom**
Fixes #13

### CODE REVIEW

File: glitter_news/admin.py [hunk 1]
```diff
@@ -35,7 +35,9 @@ def get_fieldsets(self, request, obj=None):
 
         fieldsets = (
             ('Post', {
-                'fields': ('title', 'category', 'author', 'date', 'image', 'summary', 'tags')
+                'fields': (
+                    'title', 'category', 'is_sticky', 'author', 'date', 'image', 'summary', 'tags',
```
Comment (ikonitas) [inline glitter_news/admin.py]: I don't think `is_sticky` very important I would move as the last field after the tags.

Comment (tomkins) [inline glitter_news/admin.py]: Could we move `is_sticky` to the advanced options?

Feels like an optional ignorable thing.

**Review by tomkins — COMMENTED** [review]
Happy with the rest!

File: glitter_news/admin.py (hunk 1 above)
Comment (takeontom) [reply glitter_news/admin.py]: I disagree as it's a pretty common and well used feature across all CMSs... but I'll move it. :woman_shrugging:

**Comment by takeontom** [comment]
Good to go? @tomkins @zatan

**Review by tomkins — APPROVED** [review]
:shipit: !

**Pull request merged**